syllabus/task_space/task_space.py

```diff
@@ -11,6 +11,13 @@ def __init__(self, gym_space: Union[Space, int], tasks=None):
             # Syntactic sugar for discrete space
             gym_space = Discrete(gym_space)
 
+        if isinstance(gym_space, MultiDiscrete):
+            # Syntactic sugar for discrete space
+            gym_space = MultiDiscrete(gym_space)
+            if tasks is None:
+                tasks = range(gym_space.n)
```

Comment:
This probably also needs to be a list of lists of some kind, it should be a unique name for every element of the multidiscrete